fe/fe-core/src/main/java/org/apache/doris/nereids/memo/GroupExpression.java

```diff
@@ -258,4 +263,8 @@ public String toString() {
         }
         return builder.toString();
     }
+
+    public StatsDeriveResult childStatistics(int idx) {
```

Comment:
Observe these two files and find that some code modifications are just to correct the position
look like you use `rearrange code`.
but sometimes it may be better to reduce the original code modification, it benefit cheery-pick

Reply:
Yean, I'll fix it